### PR TITLE
GUI-2306: Implement multi-select actions on snapshots landing page

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2306

Also modifies snapshots functional test sequence to allow zones to be passed as a command-line arg for https://eucalyptus.atlassian.net/browse/GUI-2430

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1551/
Test PASSed.

File: eucaconsole/views/snapshots.py [hunk 1]
```diff
@@ -79,28 +79,35 @@ def snapshots_landing(self):
...
-                    for img in images_registered:
-                        self.log_request(_(u"Deregistering image {0}").format(img.id))
-                        img.deregister()
-                    # Clear images cache
-                    self.invalidate_images_cache()
-                self.log_request(_(u"Deleting snapshot {0}").format(snapshot_id))
-                snapshot.delete()
+        if self.delete_form.validate():
+            for snapshot_id in snapshot_ids:
+                snapshot = self.get_snapshot(snapshot_id)
```
Comment (dkavanagh) [inline eucaconsole/views/snapshots.py]: This ought to be with the "with boto_error_handler..." clause

File: tests/selenium/guiops/tests/snapshots.py [hunk 2]
```diff
@@ -1,52 +1,62 @@
...
-        self.webdriver_url = parser.parse_options()['web_driver']
-        self.account = parser.parse_options()['account']
-        self.user = parser.parse_options()['user_name']
-        self.password = parser.parse_options()['password']
-        self.sauce = parser.parse_options()['sauce']
-        self.browser = parser.parse_options()['browser']
-        self.version = parser.parse_options()['version']
-        self.platform = parser.parse_options()['platform']
-        self.tester = GuiOps(console_url=self.console_url, webdriver_url=self.webdriver_url, sauce=self.sauce, browser=self.browser, version=self.version, platform=self.platform)
+        options = parser.parse_options()
```
Comment (dkavanagh) [inline tests/selenium/guiops/tests/snapshots.py]: still no base-class for common options?

Comment (kamalgill) [inline tests/selenium/guiops/tests/snapshots.py]: @dkavanagh: Agreed -- we should have a base class or method for the common options.  Will discuss with @aliceh to refactor that (probably in another ticket).

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1568/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1570/
Test PASSed.

**Pull request merged**